src/main/java/com/gluonhq/maps/MapView.java

```diff
@@ -132,6 +132,15 @@ public void setCenter(MapPoint mapPoint) {
     }
 
     /**
+     * Returns the center point of this map
+     * @return the center point
+     */
+    public MapPoint getCenter() {
+        MapPoint center = getCenter();
```

Comment:
Looks like a potential stack overflow condition. Don't think this is intentional?